cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -4499,6 +4499,111 @@
           ]
         }
       }
+    },
+    "CloudWatchDashboardSubstack": {
+      "Type": "AWS::CloudFormation::Stack",
+      "Properties": {
+        "Parameters": {
+          "CreateFSXSubstack": {
+            "Fn::If": [
+              "CreateFSXSubstack",
+              "true",
+              "false"
+            ]
+          },
+          "FSXFileSystemId": {
+            "Fn::If": [
+              "CreateFSXSubstack",
+              {
+                "Fn::GetAtt": [
+                  "FSXSubstack",
+                  "Outputs.FileSystemId"
+                ]
+              },
+              "NONE"
+            ]
+          },
+          "RAIDOptions": {
+            "Ref": "RAIDOptions"
+          },
+          "RAIDVolumesIDs": {
+            "Fn::If": [
+              "CreateRAIDSubstack",
+              {
+                "Fn::GetAtt": [
+                  "RAIDSubstack",
+                  "Outputs.Volumeids"
+                ]
+              },
+              "NONE"
+            ]
+          },
+          "CreateRAIDSubstack": {
+            "Fn::If": [
+              "CreateRAIDSubstack",
+              "true",
+              "false"
+            ]
+          },
+          "EBSVolumesIDs": {
+            "Fn::GetAtt": [
+              "EBSCfnStack",
+              "Outputs.Volumeids"
+            ]
+          },
+          "NumberOfEBSVol": {
+            "Ref": "NumberOfEBSVol"
+          },
+          "EFSFileSystemID": {
+            "Fn::If": [
+              "CreateEFSSubstack",
+              {
+                "Fn::GetAtt": [
+                  "EFSSubstack",
+                  "Outputs.FileSystemId"
+                ]
+              },
+              "NONE"
+            ]
+          },
+          "ClusterName": {
```

Comment:
For example I think it should be the first parameter.

Reply:
Done